data_validation/cli_tools.py

```diff
@@ -25,20 +25,20 @@
 data-validation connections add -c my_bq_conn BigQuery --project-id pso-kokoro-resources
 
 Step 2) Run Validation using supplied connections
-data-validation run -t Column -sc my_bq_conn -tc my_bq_conn \
+data-validation validate column -sc my_bq_conn -tc my_bq_conn \
 -tbls bigquery-public-data.new_york_citibike.citibike_trips,bigquery-public-data.new_york_citibike.citibike_stations \
 --sum '*' --count '*'
 
-python -m data_validation run -t GroupedColumn -sc my_bq_conn -tc my_bq_conn \
+python -m data_validation validate column -sc my_bq_conn -tc my_bq_conn \
 -tbls bigquery-public-data.new_york_citibike.citibike_trips \
 --grouped-columns starttime \
 --sum tripduration --count tripduration
 
-data-validation run -t Column \
+data-validation validate column \
 -sc my_bq_conn -tc my_bq_conn \
 -tbls bigquery-public-data.new_york_citibike.citibike_trips,bigquery-public-data.new_york_citibike.citibike_stations \
 --sum tripduration,start_station_name --count tripduration,start_station_name \
--rc pso-kokoro-resources.pso_data_validator.results
+-bqrh pso-kokoro-resources.pso_data_validator.results
```

Comment:
I've seen inconsistencies with the -bqrh flag on the develop branch. Do we have a test that this is currently functional?

Reply:
Yes, the --bqrh flag runs exactly the same as the -rc flag using the `cli_tools.get_result_handler()` function. We have a test for it in `test_cli_tools.py`. The code will first check to see if the --bqrh flag was supplied; if not, it will check if the --rc flag exists; If neither are provided it will set to None.